src/pages/index.tsx

```diff
@@ -251,29 +247,29 @@ const LinkCard = styled.a`
 
   p {
     text-align: left;
-    margin: ${(p) => p.theme.space[16]} 0;
+    margin: ${p => p.theme.space[16]} 0;
   }
 
-  @media (min-width: ${(p) => p.theme.breakpoints.phone}) and (max-width: 1024px) {
+  @media (min-width: ${p => p.theme.breakpoints.phone}) and (max-width: 1024px) {
     max-width: 48% !important;
   }
 
-  @media (min-width: 0) and (max-width: ${(p) => p.theme.breakpoints.phone}) {
+  @media (min-width: 0) and (max-width: ${p => p.theme.breakpoints.phone}) {
     max-width: 100% !important;
   }
 `
 
 const IconHolder = styled.span`
-  background: ${(p) => p.theme.colors.white};
+  background: ${p => p.theme.colors.white};
   box-shadow: 0px 2px 4px rgba(26, 32, 44, 0.1), 0px 3px 6px rgba(26, 32, 44, 0.05);
   border-radius: 50%;
   height: 56px;
   width: 56px;
   display: flex;
   align-items: center;
   justify-content: center;
-  @media (min-width: 0) and (max-width: ${(p) => p.theme.breakpoints.phone}) {
-    margin: ${(p) => p.theme.space[16]} ${(p) => p.theme.space[24]};
+  @media (min-width: 0) and (max-width: ${p => p.theme.breakpoints.phone}) {
+    margin: ${p => p.theme.space[16]} ${p => p.theme.space[24]};
```

Comment:
A lot of changes like this in this PR which somehow looks like prettier configurations fighting.

Reply:
They're all applied by the pre-commit hook and I don't have any conflicting local Prettier file 🤷 I'll check with Nilu